packages/react-native/template/_gitignore

```diff
@@ -21,6 +21,7 @@ DerivedData
 *.ipa
 *.xcuserstate
 ios/.xcode.env.local
+visionos/.xcode.env.local
```

Comment:
Maybe this?
```suggestion
**/.xcode.env.local
```

Reply:
I think in that form it can be upstreamed to core, nice catch!

Reply:
https://github.com/facebook/react-native/pull/42963